### DISCUSSION TITLE
v4.6.0

### DISCUSSION THREAD

**Discussion by Keboo** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2862] [Announcements]
### Key Changes
* Increasing minimum framework to net462 by @Keboo in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2663
* Adding new TimeChanged event to Clock by @Keboo in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2716
* Add virtualization on ComboBoxes and DataGridComboBoxColumn by @Xaalek in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2728
* Added MD3 version of MaterialDesignSwitchToggleButton by @jizc in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2748
* Remove ripple effect on filled fields by @Xaalek in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2749
* Added UniformCornerRadius to Flipper by @nicolaihenriksen in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2758
* Split Card template into 2 styles: Elevated and Outlined by @nicolaihenriksen in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2757
* Refactor fix for 2689 to use attached properties by @nicolaihenriksen in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2762
* Various changes for cards and shadows by @Xaalek in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2765
* Allow "half" rating selection by @nicolaihenriksen in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2764
* Fix TabControl styles by @Xaalek in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2779
* Replaced ShadowDepth with Elevation by @jizc in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2786
* Added BorderThickness attached DP for outlined date/time pickers by @nicolaihenriksen in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2805
* New feature - PasswordBox styles with "reveal password" functionality by @nicolaihenriksen in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2828
* [PackIcon changes](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/4-x-icon-changes#pack-icon-changes-450--460)

## New Contributors
* @cjmurph made their first contribution in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2692
* @nicolaihenriksen made their first contribution in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2758
* @michaldivis made their first contribution in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2795
* @binarycow made their first contribution in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2807
* @dependabot made their first contribution in https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2856

**Full Changelog**: https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/compare/v4.5.0...v4.6.0

A big thank you to everyone who contributed (either with issues or pull requests):
@avanbremen
@bilaxos
@binarycow
@cjmurph
@DenZuck
@DrCQ
@dudley810
@Erapchu
@hfann
@jamesport079
@JDeuker
@jizc
@lszczygielek
@michaldivis
@mikechristiansenvae
@NeilMacMullen
@nicolaihenriksen
@reiseder
@TinoMSoft
@washburnit
@wj8400684
@Xaalek

<hr /><em>This discussion was created from the release <a href='https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/releases/tag/v4.6.0'>v4.6.0</a>.</em>

**Comment by ElieTaillard** [comment]
What happened to the demo app ? Panels are not wrapping anymore.

**Reply by Keboo** [reply]
Bug on my part. I noticed it right after doing the release. Just put in a PR for it #2866.

**Reply by ElieTaillard** [reply]
It might be worth releasing a 4.6.1 version soon.

**Reply by Keboo** [reply]
Agreed. Though since it is just the demo app I might cheat and just update the demo app on the release.

**Reply by Keboo** [reply]
Actually I think you are right with a 4.6.1 release. Will get something out in the next day or two.

**Reply by Keboo** [reply]
Just pushed out 4.6.1
